frontend/src/components/ide/AIAssistant/components/PendingChanges.jsx

```
@@ -0,0 +1,221 @@
+            change.status === 'pending' ? "border-yellow-500/30 bg-yellow-500/5" : "",
+            change.status === 'applying' ? "border-blue-500/30 bg-blue-500/5" : "",
+            change.status === 'applied' ? "border-green-500/30 bg-green-500/5" : "",
+            change.status === 'rejected' ? "border-red-500/30 bg-red-500/5 opacity-50" : ""
+        )}>
+            {/* Header */}
+            <div
+                className="flex items-center gap-2 p-3 cursor-pointer hover:bg-muted/30"
+                onClick={() => setIsExpanded(!isExpanded)}
+            >
+                {/* Expand/Collapse */}
+                <button className="p-0.5">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)